Feature/add availability

michael-kamel opened the pull request:
Closes #8

chaoran-chen commented on src/test/java/science/icebreaker/device_availability/DeviceAvailabilityServiceTest.java:
```diff
@@ -0,0 +1,107 @@
+    public DeviceAvailabilityServiceTest(DeviceAvailabilityService deviceAvailabilityService,
+            AccountService accountService, DeviceAvailabilityRepository deviceAvailabilityRepository,
+            WikiPageRepository wikiPageRepository, AccountRepository accountRepository)
+            throws AccountCreationException {
```
```suggestion
            WikiPageRepository wikiPageRepository, AccountRepository accountRepository) {
```

chaoran-chen commented on src/main/java/science/icebreaker/device_availability/DeviceAvailabilityService.java:
```diff
@@ -0,0 +1,69 @@
+     * Adds the device availability entry
+     * 
+     * @param deviceData The device availability data
+     * @return The ID of the availability entry.
```
The documentation does not match the code.

chaoran-chen reviewed (commented):
Looks good :) I just have two tiny comments.

And please rebase it.

michael-kamel committed:
add adding device availability

fix swagger docs & add additional method docs

update src/test/java/science/icebreaker/device_availability/DeviceAvailabilityServiceTest.java

Co-authored-by: Chaoran Chen <mail@chaoran-chen.de>

fix addDeviceAvailability doc & remove unused import
